Helper/Data.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Dotdigitalgroup\Enterprise\Helper;
+
+use Magento\Framework\App\Helper\Context;
+use Dotdigitalgroup\Enterprise\Helper\Config;
```

Comment:
@adeelq I think you can remove this line :

use Dotdigitalgroup\Enterprise\Helper\Config;